src/main/java/gov/cabinetoffice/gap/adminbackend/services/GrantAdvertService.java

```diff
@@ -180,9 +181,12 @@ public void updatePageResponse(GrantAdvertPageResponseValidationDto pagePatchDto
         GrantAdvert grantAdvert = grantAdvertRepository.findById(pagePatchDto.getGrantAdvertId())
                 .orElseThrow(() -> new NotFoundException(
                         String.format("GrantAdvert with id %s not found", pagePatchDto.getGrantAdvertId())));
+
+        advertPublishedOrScheduledError(grantAdvert);
```

Comment:
Thinking this could be more readable, should describe what the method is doing i.e validateAdvertStatus

Reply:
Good idea, fixed in latest